src/qmltypes/qmlfilter.cpp

```diff
@@ -259,7 +259,17 @@ int QmlFilter::savePreset(const QStringList &propertyNames, const QString &name)
             dir.cd(objectNameOrService());
     }
     const QString preset = name.isEmpty()? tr("(defaults)") : name;
-    properties.save(dir.filePath(preset).toUtf8().constData());
+    // Convert properties to YAML string.
+    char* yamlStr = properties.serialise_yaml();
+    QString yaml = yamlStr;
+    free(yamlStr);
+    // Save YAML to file
+    QFile yamlFile(dir.filePath(preset));
+    if(!yamlFile.open(QIODevice::WriteOnly)) {
+        qDebug() << "Failed to save preset: " << dir.filePath(preset);
```

Comment:
Use LOG_ERROR() instead of qDebug() as that uses our logging framework.
It would be nice at this point to let the user know, but from this point in the code that is difficult. :-(

Reply:
done.